app/controllers/admin/groups_controller.rb

```diff
@@ -0,0 +1,65 @@
+class Admin::GroupsController < Admin::BaseController
+
+  def index
+    authorize :groups, :index?
```

Comment:
There is no need to put the `:index?` here. The action which is authorized is deduced from the action name. If you are in the `index` method and you call `authorize :groups` it will authorize against `:index?`